service/controller/clusterapi/v29/cluster_resource_set.go

```diff
@@ -614,7 +600,6 @@ func NewClusterResourceSet(config ClusterResourceSetConfig) (*controller.Resourc
 		tccpRouteTablesResource,
 		tccpSubnetsResource,
 		regionResource,
-		asgStatusResource,
```

Comment:
Dropping from the cluster controller because this is TCCP and the ASG is now in TCNP.